data/common.yaml

```diff
@@ -32,6 +32,10 @@ gdm::packages:
   liberation-mono-fonts: ~
   liberation-sans-fonts: ~
   liberation-serif-fonts: ~
+  xorg-x11-utils: ~
+  xorg-x11-docs: ~
+  orca: ~
```

Comment:
`orca` was not previously a required package for el7.  Has that changed?

Reply:
orca is requirement in gnome.  I must have gotten the two list mixed up.  Removed orca from the rpm list.  It is installed if you also include the pupmod-simp-gnome module.